### PR TITLE
Added the Ropsten certificate store of Lasalle and NAFA

### PR DESCRIPTION
**Pull request by skydudie**
Editing the registry to include Lasalle's and NAFA's Ropsten certificate store addresses

### CODE REVIEW

**Comment by skydudie** [comment]
@Nebulis I've also added the registry entries for SMU Academy

**Comment by Nebulis** [comment]
I received details for Lasalle but not for the rest, if you create another MR with Lasalle details only I will be able to proceed to merge it

**Comment by skydudie** [comment]
@Nebulis I've removed the entries for NAFA and SMU Academy. This PR only contains Lasalle's Ropsten and Mainnet cert stores

File: static/registry.json [hunk 1]
```diff
@@ -224,6 +233,10 @@
     "0x146d1e1938dc0f5fbb2179727c2fa61522cd1834": {
       "name": "ROPSTEN: National Institute of Early Childhood Development",
       "displayCard": false
-    }
+    },
+	"0xb02bcBb4865d0301F6e4A5f46043498f7c348E03": {
+      "name": "ROPSTEN: Lasalle College of the Arts",
+      "displayCard": false
+    },
```
Comment (Nebulis) [inline static/registry.json]: need to remove this comma https://travis-ci.org/github/OpenCerts/opencerts-website/jobs/663004116?utm_medium=notification&utm_source=github_status

**Pull request merged**